### PR TITLE
Use `docker compose` to run locally

### LINKED ISSUE #8: Use docker compose to run locally
Instead of only running backend in containers, a [docker-compose](https://docs.docker.com/compose/) should be more accessible and easier to the end-user, since they will need docker to run the backend anyway. That should even help when developing frontend.

What needs to be done in order to achieve it:  
* Create a dockerfile that builds the project and then runs it.
* Create a docker-compose file that runs both the backend and then the frontend, all in the same network. In the future, the backend application can be in more than one network, to be in a separate network with only backend app and frontend. 

With this, you should be able to run all applications (in the proper order) with `docker-compose up`, creating a docker container for each service instead of doing that with separate commands and the application in the shell session.

### PR DESCRIPTION
**Pull request by Henriquelay**
This solution also fixes some stuff going on with envs, and uses Docker Compose V2. Be sure to have that installed.

### CODE REVIEW

**Comment by AtilioA** [comment]
:8ball:

**Comment by AtilioA** [comment]
(achievement unlocked)

File: compose.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    depends_on:
+      - postgres
+      - redis
+    working_dir: /app
+    # TODO transpile and serve
+
+  postgres:
+    image: postgres:alpine
+    environment:
+      POSTGRES_USER: alertadotesouro
```
Comment (AtilioA) [inline compose.yaml:104]: I'm assuming the user needs to edit the .yaml file to change what is used for Postgres credentials?

Comment (AtilioA) [reply compose.yaml:104]: This occurs in other places as well; as long as the values match the ones from the .env.example, it shouldn't be much of a problem.

**Review by AtilioA — COMMENTED** [review]
Nice work on the configs and docker-compose :handshake: 🙌 I hope it works as intended soon enough. Let's hold off on the merge until we make sure it is working on both of our machines without hacks.

**Comment by AtilioA** [comment]
perdi tncc

File: compose.yaml (hunk 1 above)
Comment (Henriquelay) [reply compose.yaml:104]: Yes, we urgently need a use envs correctly issue

**Pull request merged**